doc/samples/readme-extended-UWP-samples.md

```diff
@@ -0,0 +1,74 @@
+# Extended UWP Samples
+
+## Overview
+
+Extended UWP Samples solution (InteropSamples.sln) contains 3 projects:
```

Comment:
You recommend to use something (InteropSamples.sln) which is not under CI testing. We do not actually verify that this solution is actually functional. Why we can't direct everyone to use AllBuild.sln as we already did for tutorials?

Reply:
I think we should add it to CI than. This sln is essential for providing UWP samples as standalone package.